### PR TITLE
Feature/account review card

### PR DESCRIPTION
**Pull request by KShih**
1. display review history in account page
2. add link to that review page(location/location.id/)

image

### CODE REVIEW

File: mainapp/templates/account.html [hunk 1]
```diff
@@ -15,7 +21,46 @@ <h1>Account</h1>
...
+        <div class="col">
+        {% if user.review_set %}
+            <h1> Submitted Review: </h1>
+
+            <div class="mb-3 text-center location-results">
+                {% for review in user.review_set.all %}
+                <div class="card mb-4 shadow-sm "style="max-width: 27rem;">
+                    <div class="card-header">
+                        <ul class="list-unstyled">
+                            <a href="/location/{{review.location.id}}/">{{ review.location.address }}</a>
```
Comment (alldne) [inline mainapp/templates/account.html]: I think this way is safer and django-ish:
```
<a href="{% url 'location' location.id %}">
```

Comment (KShih) [reply mainapp/templates/account.html]: @alldne updated

File: mainapp/templates/account.html [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+{% endblock %}
+{% load static %}
 {% block content %}
+<br>
 <div class="container">
     <div class="row justify-content-center">
-        <div class="col-7">
+
+        <div class="col col-7 ">
```
Comment (mok232) [inline mainapp/templates/account.html]: I think the width of the account pane should be <= width of reviews pane

File: mainapp/templates/account.html [hunk 3]
```diff
@@ -15,7 +21,46 @@ <h1>Account</h1>
                 <li class="list-group-item">User type: {{ user.user_type_string }}</li>
             </ul>
         </div>
+        <div class="col">
+        {% if user.review_set %}
+            <h1> Submitted Review: </h1>
+
+            <div class="mb-3 text-center location-results">
```
Comment (mok232) [inline mainapp/templates/account.html]: Review pane width should be larger

Comment (KShih) [reply mainapp/templates/account.html]: image
Does it look better?

Comment (Ayushi-0903) [reply mainapp/templates/account.html]: Yes it looks better.

Comment (KShih) [reply mainapp/templates/account.html]: @mok232 updated.

Comment (mok232) [reply mainapp/templates/account.html]: LGTM!

**Pull request merged**